crates/wit-component/tests/components/error-link-duplicate-symbols/error.txt

```diff
@@ -0,0 +1,114 @@
+duplicate symbol(s): [
```

Comment:
Could this, and some of the other errors here, perhaps be formatted without `Debug` output?